csrankings-8.csv

```diff
@@ -538,7 +538,7 @@ Song Jiang,University of Texas at Arlington,http://ranger.uta.edu/~sjiang,vhpbMX
 Song Min Kim,KAIST,https://sites.google.com/view/smilelab,lBJc3B8AAAAJ
 Song Wang 0002,University of South Carolina,http://www.cse.sc.edu/~songwang,eycXl_QAAAAJ
 Song Wang 0009,York University,https://www.eecs.yorku.ca/~wangsong,gzyZhcgAAAAJ
-Song-Chun Zhu,Univ. of California - Los Angeles,http://www.stat.ucla.edu/~sczhu,Al8dyb4AAAAJ
+Song-Chun Zhu,Peking University,http://www.stat.ucla.edu/~sczhu,Al8dyb4AAAAJ

```

Comment:
No indication on this website that this faculty member has moved.

Reply:
Thank you for your response, here is the official news from Peking University: [Link](http://newsen.pku.edu.cn/news_events/news/media/10191.htm)

The [wiki page](https://en.wikipedia.org/wiki/Song-Chun_Zhu) is also updated. (Under Career section: *In September 2020, Zhu returned to China to join Peking University...* )

This pull request is from the official github account of the lab (I am a PhD student supervised by the professor, and one of the admin of the account).

The homepage is still under UCLA for now, and we may update the link in the future when the new page is ready.

Reply:
@emeryberger The prof has also updated the [homepage](http://www.stat.ucla.edu/~sczhu/):
`Institute for Artificial Intelligence, Peking University, Since Nov. 2020` (under **Director of** section).

If you need more information to merge the PR, feel free to let me know.